configs/_base_/datasets/ade20k_512x512.py

```diff
@@ -0,0 +1,54 @@
+# dataset settings
```

Comment:
I think we can use this cfg file https://github.com/open-mmlab/mmsegmentation/blob/master/configs/_base_/datasets/ade20k.py

Reply:
The two config files are different, `w.r.t` to the settings of `Resize`

Reply:
Maybe this is an unfair comparison (training size (2560, 512) large than (2048, 512)). I suggest using the (2048, 512) cfg file.

Reply:
Need to modify config name like BEiT

Reply:
> Maybe this is an unfair comparison (training size (2560, 512) large than (2048, 512)). I suggest using the (2048, 512) cfg file.

Fixed